src/pavan/PavanB_coding_exam_1/AddIndices.java

```diff
@@ -0,0 +1,30 @@
+package pavan.PavanB_coding_exam_1;
+
+import java.util.Arrays;
+
+public class AddIndices {
+	
+	void getSumIndices(int[]input, int target){
+
+		int[]result= new int[2];
+
+
+		for(int index=0; index<input.length;index++) {
+			for (int innerindex=1;innerindex<input.length;innerindex++) {
```

Comment:
Inner for loop should from i+1, think on this, if you want to connect on Skype then feel free to message.

Take reference of this code
Kangan - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/987/files
Rupali - https://github.com/KrishnaKTechnocredits/JAVATechnoMar2021/pull/991/files